src/editor/editorinteractive.cc

```diff
@@ -584,17 +597,17 @@ void EditorInteractive::think() {
 	egbase().get_gametime_pointer().increment(Duration(realtime_ - lasttime));
 }
 
-void EditorInteractive::exit() {
-	if (need_save_) {
-		if ((SDL_GetModState() & KMOD_CTRL) != 0) {
-			end_modal<UI::Panel::Returncodes>(UI::Panel::Returncodes::kBack);
-		} else {
-			UI::WLMessageBox mmb(this, UI::WindowStyle::kWui, _("Unsaved Map"),
-			                     _("The map has not been saved, do you really want to quit?"),
-			                     UI::WLMessageBox::MBoxType::kOkCancel);
-			if (mmb.run<UI::Panel::Returncodes>() == UI::Panel::Returncodes::kBack) {
-				return;
-			}
+void EditorInteractive::exit(const bool force) {
+	if (force) {
+		end_modal<UI::Panel::Returncodes>(UI::Panel::Returncodes::kBack);
+	} else {
```

Comment:
```suggestion
	if (!force) {
```

Reply:
I just copied the old KMOD_CTRL branch, though I wondered why this is called twice. If you say it's OK to remove it, I'll do it.